tubular/misc.py

```diff
@@ -74,8 +76,31 @@ def __init__(
 
         super().__init__(columns=columns, **kwargs)
 
-    @nw.narwhalify
-    def transform(self, X: FrameT) -> FrameT:
+    @block_from_json
+    def to_json(self) -> dict[str, dict[str, Any]]:
+        """Dump transformer to json dict.
+
+        Returns
+        -------
+        dict[str, dict[str, Any]]:
+            jsonified transformer. Nested dict containing levels for attributes
+            set at init and fit.
+
+        Examples
+        --------
+        >>> transformer = SetValueTransformer(columns='a', value=1)
+        >>> transformer.to_json()
+        {'tubular_version': 'dev', 'classname': 'SetValueTransformer', 'init': {'columns': ['a'], 'copy': False, 'verbose': False, 'return_native': True, 'value': 1}, 'fit': {}}
```

Comment:
could you change `'dev'` to `...` here? tubular version will be detected differently depending on whether the code is run locally or in CI, so this is more flexible and will pass in both contexts :)
```suggestion
        {'tubular_version': ..., 'classname': 'SetValueTransformer', 'init': {'columns': ['a'], 'copy': False, 'verbose': False, 'return_native': True, 'value': 1}, 'fit': {}}
```